### PR TITLE
Restore functionality of `include_non_prod`

### PR DESCRIPTION
**Pull request by tommyjcarpenter**

### CODE REVIEW

File: merqube_client_lib/api_client/base.py [hunk 1]
```diff
@@ -108,8 +108,7 @@ def get_index_defs(
         """
 
         endpoint = "/index"
-        if include_nonprod:
-            endpoint += "?type=all"
+        endpoint += "?stage=all" if include_nonprod else "?stage=prod"
```
Comment (palmserino) [inline merqube_client_lib/api_client/base.py:111]: Don't we not need `?stage=all` if `include_nonprod`?

Might be cleaner to do:
`if include_nonprod`
` endpoint += "?stage=prod`

**Review by palmserino — COMMENTED** [review]
Do we need that weird `log.txt` file? Is that maybe from local testing?

File: merqube_client_lib/api_client/base.py (hunk 1 above)
Comment (tommyjcarpenter) [reply merqube_client_lib/api_client/base.py:111]: yes, could remove that (harmless but will remove)

**Review by palmserino — APPROVED** [review]
`/index?stage` stuff LGTM - not too familiar with everything else

**Pull request merged**